mat64/dense.go

```diff
@@ -512,7 +512,13 @@ func (m *Dense) UnmarshalBinary(data []byte) error {
 	m.mat.Stride = int(cols)
 	m.capRows = int(rows)
 	m.capCols = int(cols)
-	m.mat.Data = use(m.mat.Data, m.mat.Rows*m.mat.Cols)
+
```

Comment:
Alternatively, there could be a limit on the number of elements less than or equal to MaxInt32. `gob` has 1GB of memory limit, MaxInt32 of float64 is 16GB of memory.